topology/probes/k8s/node.go

```diff
@@ -116,9 +116,20 @@ func (c *nodeProbe) OnDelete(obj interface{}) {
 	}
 }
 
+func (c *nodeProbe) OnNodeAdded(n *graph.Node) {
+	hostname, _ := n.GetFieldString("Hostname")
+	if t, _ := n.GetFieldString("Type"); t == "host" {
```

Comment:
this condition is redundant, your indexer only triggers events when `"Type" == "host"`

Reply:
It was a precaution but a useless one. Removed